webxrlayers-1.bs

```diff
@@ -147,8 +147,8 @@ Application flow {#applicationflow}
 
 If an author wants to use layers, they have to go through these steps:
  1. request support through {{XRPermissionDescriptor/requiredFeatures}} or {{XRPermissionDescriptor/optionalFeatures}} in {{XRSystem/requestSession()}}
- 1. create a XRWebGLLayerFactory or XRMediaLayerFactory
- 1. create layers with these factories
+ 1. create a {{XRWebGLBinding}} or {{XRMediaLayerFactory}}
```

Comment:
Should we also rename `XRMediaLayerFactory` -> `XRMediaBinding`? It's a bit different in that it's not really "binding" to anything at creation time (except maybe the DOM?) but it seems like it would be ideal to use the same terminology for the same basic concept of supplying imagery to/from the session.

Reply:
yes. I will make that change.